fe/src/pages/Insights/InsightTable.tsx

```diff
@@ -122,7 +122,7 @@ const InsightTable: FC = () => {
 									{row.type}
 								</TableCell>
 								<TableCell sx={tableStyles.bodyRow} align="left">
-									{row.id}
+									{row.displayId || row.id}
```

Comment:
The DisplayId field is empty when no specific display ID has been set on a resource (falls back to the Id). Do we want to keep this behavior or should DisplayId always be set?

Reply:
I think we should only one ID column but with a if condition on the value:
- if display is set: show display ID
- otherwise show id
- it looks like this is what you have done